src/components/PersonLink.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person,
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => (
+  <Link
+    to={`/people/${person.slug}`}
+    className={person.sex === 'f' ? 'has-text-danger' : ''}
```

Comment:
Create a enum for magic word 'f' and use it everywhere.
Also use a classnames library for add classes with condition